src/cpp/server/ServerPAMAuth.cpp

```diff
@@ -162,15 +130,15 @@ void doSignIn(const http::Request& request,
                                                             &plainText);
       if (error)
       {
-         LOG_ERROR(error);
+         LOG_ERROR_MESSAGE("Failed sign-in - unable to decrypt password - error: " + error.asString());
```

Comment:
Here and elsewhere - I think we'll lose some of the error metadata if we don't log the error directly so we should add these as properties.
```suggestion
         error.addDescription("Failed sign-in - unable to decrypt password");
         LOG_ERROR(error);
```